recipe/meta.yaml

```diff
@@ -21,12 +21,13 @@ requirements:
     - pip
   run:
     - python
-    - ipykernel >=4.1
-    - jupyter_client >=4.1
-    - jupyter_core
-    - pygments
     - pyqt
     - traitlets
```

Comment:
`pyqt` is not listed in the `install_requirements` but I guess it is still need. Leaving it here.

Reply:
qtconsole won't work without some Qt bindings installed. They are not listed in `install_requirements` to allow pip users decide what bindings they want to use.

But since Anaconda/conda-forge only supports PyQt5 for now, then the requirement is valid. Else, you'll run

    conda install -c conda-forge qtconsole

and you'll get a broken packages.